src/power_grid_model_io/functions/filters.py

```diff
@@ -31,8 +31,8 @@ def exclude_value(row: pd.Series, col: str, value: Union[float, str]) -> bool:
     if col not in row:
         raise ValueError(f"The column: '{col}' cannot be found for the filter")
     result = row[col] != value
-    if isinstance(result, pd.Series):
-        return result.item()
+    if isinstance(result, pd.Series):  # sonar-ignore: S2583
+        return result.item()  # sonar-ignore: S2583
```

Comment:
please attempt to mark only one of the two and provide a comment as to why you ignore this (false positive)